src/ext/java/no/entur/uttu/ext/entur/stopplace/EnturMummuStopPlaceRegistry.java

```diff
@@ -134,4 +136,9 @@ public boolean currentValidityFilter(EntityInVersionStructure entity) {
           .isAfter(validBetween.getToDate().atZone(ZoneId.systemDefault()).toInstant())
       );
   }
+
+  @Override
+  public List<StopPlace> getAllStopPlaces() {
+    return List.of();
```

Comment:
Would it be okay to keep it like this for now?

Reply:
Yes